Sources/AppcuesKit/Push/PushMonitor.swift

```diff
@@ -65,6 +73,25 @@ internal class PushMonitor: PushMonitoring {
         refreshPushStatus(publishChange: true)
     }
 
+    func configureAutomatically() {
```

Comment:
should this function set `configuredAutomatically` true?

Reply:
Indeed it should!